### PR TITLE
remove enableBatch from spine/db

### PR DESCRIPTION
**Pull request by zhakesi**
清除spine/dragonbones模块关于合批的相关代码.避免用户设置后无效或出现其他问题.

### CODE REVIEW

File: cocos/spine/assembler/simple.ts [hunk 1]
```diff
@@ -261,31 +257,19 @@ function updateComponentRenderData (comp: Skeleton, batcher: Batcher2D) {
...
-    _handleVal = 0x00;
     _needColor = false;
     _vertexEffect = comp._effectDelegate && comp._effectDelegate._vertexEffect as any;
 
     if (nodeColor._val !== 0xffffffff ||  _premultipliedAlpha) {
         _needColor = true;
     }
 
-    if (_useTint) {
-        _handleVal |= FLAG_TWO_COLOR;
```
Comment (pandamicro) [inline cocos/spine/assembler/simple.ts:273]: Shouldn't remove this

File: cocos/spine/skeleton.ts [hunk 2]
```diff
@@ -438,24 +436,6 @@ export class Skeleton extends Renderable2D {
...
-    //     if (value != this._enableBatch) {
-    //         this._enableBatch = value;
-    //         this._updateBatch();
-    //     }
-    // }
-
-    // @serializable
-    // private _enableBatch: boolean = true;
-
-    public enableBatch = false;
```
Comment (pandamicro) [inline cocos/spine/skeleton.ts:458]: Please add to deprecated-v3.5.1.ts

File: cocos/dragon-bones/ArmatureDisplay.ts [hunk 3]
```diff
@@ -421,12 +419,6 @@ export class ArmatureDisplay extends Renderable2D {
     protected _debugBones = false;
     /* protected */ _debugDraw: Graphics | null = null;
 
-    /**
-     * @deprecated since v3.5.0, this is an engine private interface that will be removed in the future.
-     */
-    @serializable
-    public _enableBatch = false;
```
Comment (pandamicro) [inline cocos/dragon-bones/ArmatureDisplay.ts:428]: Please add to deprecated-v3.5.1.ts

**Review by pandamicro — CHANGES_REQUESTED** [review]

File: cocos/spine/assembler/simple.ts (hunk 1 above)
Comment (zhakesi) [reply cocos/spine/assembler/simple.ts:273]: > Shouldn't remove this

no used _handleVal  with FLAG_TWO_COLOR. use _useTint flag directly.

**Review by pandamicro — APPROVED** [review]

**Pull request merged**